File: Leetcode/Leet1431.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int max = candies[0];
        List<Boolean> ans = new ArrayList<Boolean>();
        for(int i = 1; i< candies.length; i++){
            if( candies[i] > max){
                max = candies[i];
            }
        }
        for(int i = 0; i< candies.length; i++){
            if((candies[i]+extraCandies) >= max){
                ans.add(true);
            }
            else{
                ans.add(false);
            }
        }
        return ans;
    }
}

File: Leetcode/Leet1431.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        mx = max(candies)
        arr = []
        for i in range(len(candies)):
            if candies[i] + extraCandies >= mx:
                arr.append(True)
            else:
                arr.append(False)
        return arr
        